### DISCUSSION TITLE
Miglior libreria

### DISCUSSION THREAD

**Discussion by berta24** [MainKronos/Sonarr-AnimeDownloader#63] [General]
Io personalmente utilizzo Plex Plus, ho codificato il path per gli anime come libreria anime, un path per film e serie tv. Utilizzo plex plus perchè ha molte funzioni interessanti tipo il discovery che ti permette di vedere serie tv da  tutte le piattaforme collegate oppure lo skip intro.

**Comment by ManuJapan89** [comment]
plex plus intendi plex con il plex pass? se si, io ho il lifetime pass

**Reply by berta24** [reply]
Si intendevo il plex pass

**Comment by MainKronos** [comment]
Uso anche io plex da un po' di anni ma non ho mai comprato plex plus.
Ero intenzionato a comprarlo per utilizzarlo anche sul tablet, perchè senza il plus non è possibile usarlo sui dispositivi mobili.
Dite che ne vale la pena?

**Reply by ManuJapan89** [reply]
puoi pagare 5 euro di attivazione una tantum su device android/ios. e lo puoi usare senza il pass
ne vale la pena per la roba in piu. skip intro .hw trascoding(gpu) e download su mobile appunto
ma ultimamente sono piu i problemi che altro.
emby sembra bello ma non ha il cestino, se una porzione dei file sparisce, esempio un hdd esterno và down...lui li toglie dalla libreria e se rimetti su il disco lui riscansiona e devi rimatchare tutto da capo. è improponibile

**Reply by berta24** [reply]
Si esatto, io ho fatto l'attivazione una tantum perché lo usavo parecchio da telefono, poi ho voluto provare il pass quando è uscita la funzione discovery, e l'ho tenuto per tutto il resto
